### PR TITLE
Update intro sections

### PR DESCRIPTION
**Pull request by noon-dawg**
## Summary

This pull request updates the text in the introduction sections to follow the writing style.

## Links

N/A

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/description.md [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
-It has predictable resource URLs.
-It returns HTTP response codes to indicate errors.
+The Rebilly API is built on HTTP and is RESTful.
+It has predictable resource URLs and returns HTTP response codes to indicate errors.
 It also accepts and returns JSON in the HTTP body.
-You can use your favorite HTTP/REST library for your programming language to use the API,
-or you can use one of our SDKs (currently available in [PHP](https://github.com/Rebilly/rebilly-php) and [Javascript](https://github.com/Rebilly/rebilly-js-sdk)).
+Use your favorite HTTP/REST library in your programming language when using this API,
+or use one of the Rebilly SDKs,
+which are available in [PHP](https://github.com/Rebilly/rebilly-php) and [Javascript](https://github.com/Rebilly/rebilly-js-sdk).
```
Comment (ashkarpetin) [inline openapi/description.md:8]:
```suggestion
which are available in [PHP](https://github.com/Rebilly/rebilly-php) and [JavaScript](https://github.com/Rebilly/rebilly-js-sdk).
```

File: openapi/openapi.yaml [hunk 2]
```diff
@@ -62,10 +62,9 @@ components:
 
     PublishableApiKey:
       description: |-
-        Only for the Tokens resource.
+        This authentication method is only applicable for the tokens resource.
```
Comment (adamaltman) [inline openapi/openapi.yaml:65]:
```suggestion
        This authentication method is applicable to specific operations.
```

There are now about 20 API operations that use this method.

File: openapi/description.md [hunk 3]
```diff
@@ -53,23 +51,24 @@ Rebilly follows the error response format proposed in [RFC 7807](https://tools.i
...
 
-Rebilly also offers [FramePay](https://docs.rebilly.com/docs/developer-docs/framepay/),
- a client-side iFrame-based solution to help
-create payment tokens while minimizing PCI DSS compliance burdens
-and maximizing the customization ability. [FramePay](https://docs.rebilly.com/docs/developer-docs/framepay/)
-is interacting with the [payment tokens creation operation](https://all-rebilly.redoc.ly/tag/Payment-tokens/operation/PostToken/).
+Rebilly also provides [FramePay](https://docs.rebilly.com/docs/developer-docs/framepay/),
+a client-side iFrame-based solution, to help create payment tokens while minimizing PCI DSS compliance burdens
+and maximizing your customization ability.
+[FramePay](https://docs.rebilly.com/docs/developer-docs/framepay/) interacts with the [payment tokens creation operation](https://all-rebilly.redoc.ly/tag/Payment-tokens/operation/PostToken).
```
Comment (adamaltman) [inline openapi/description.md:60]: This link to docs.rebilly.com is outdated. 
```suggestion
[FramePay](https://www.rebilly.com/docs/developer-docs/framepay/) interacts with the [payment tokens creation operation](https://all-rebilly.redoc.ly/tag/Payment-tokens/operation/PostToken).
```

File: openapi/description.md [hunk 4]
```diff
@@ -53,23 +51,24 @@ Rebilly follows the error response format proposed in [RFC 7807](https://tools.i
...
-the API.  However, no SDK is required to use the API.
+Rebilly provides a JavaScript SDK and a PHP SDK to help interact with the Rebilly API.
+However, no SDK is required to use the API.
 
-Rebilly also offers [FramePay](https://docs.rebilly.com/docs/developer-docs/framepay/),
- a client-side iFrame-based solution to help
-create payment tokens while minimizing PCI DSS compliance burdens
-and maximizing the customization ability. [FramePay](https://docs.rebilly.com/docs/developer-docs/framepay/)
-is interacting with the [payment tokens creation operation](https://all-rebilly.redoc.ly/tag/Payment-tokens/operation/PostToken/).
+Rebilly also provides [FramePay](https://docs.rebilly.com/docs/developer-docs/framepay/),
```
Comment (adamaltman) [inline openapi/description.md:57]:
```suggestion
Rebilly also provides [FramePay](https://www.rebilly.com/docs/developer-docs/framepay/),
```

File: openapi/description.md [hunk 5]
```diff
@@ -79,65 +78,73 @@ $client = new Rebilly\Client([
...
-- You can use gte (greater than or equals) filter like this: `?filter=amount:1..`, or lte (less than or equals) than filter like this: `?filter=amount:..10`.
+- To use a gte (greater than or equals) filter, use: `?filter=amount:1..`.
+  This also works for datetime-based fields.
+  
+- To use a lte (less than or equals) filter, use: `?filter=amount:..10`.
   This also works for datetime-based fields.
 
-- You can create some [specified values lists](https://all-rebilly.redoc.ly/#tag/Lists) and use them in filter: `?filter=firstName:@yourListName`.
-  You can also exclude list values: `?filter=firstName:!@yourListName`.
+- To create [specified values lists](https://user-api-docs.rebilly.com/#tag/Lists) and use them in filters, use: `?filter=firstName:@yourListName`. \
```
Comment (adamaltman) [inline openapi/description.md:109]: Link to all api instead.

Comment (adamaltman) [reply openapi/description.md:109]:
```suggestion
- To create [specified values lists](https://all-rebilly.redoc.ly/#tag/Lists) and use them in filters, use: `?filter=firstName:@yourListName`. \
```

**Review by adamaltman — APPROVED** [review]

**Comment by adamaltman** [comment]
Side note: these will eventually be eliminated when we remove the product bundler.

**Review by swilczynski — APPROVED** [review]

**Pull request merged**